signature-aggregator/aggregator/aggregator.go

```diff
@@ -179,12 +190,71 @@ func (s *SignatureAggregator) CreateSignedMessage(
 		return nil, err
 	}
 
-	accumulatedSignatureWeight := big.NewInt(0)
+	isL1 := false
+	if signingSubnet != constants.PrimaryNetworkID {
+		subnet, err := s.pChainClient.GetSubnet(context.Background(), signingSubnet, s.pChainClientOptions...)
+		if err != nil {
+			s.logger.Error(
+				"Failed to get subnet",
+				zap.String("signingSubnetID", signingSubnet.String()),
+				zap.Error(err),
+			)
+			return nil, err
+		}
+		if subnet.ConversionID != ids.Empty {
+			isL1 = true
+		}
```

Comment:
```suggestion
		isL1 = subnet.ConversionID != ids.Empty
```

Also not required for this PR, but we may want to make this check for given source/destination chains once on startup, or maybe periodically after for non-converted subnets, rather than every time we try to construct an aggregate signature. 

If it ever is `true`, we know if will always be true in the future, so we can avoid the extra calls.

Reply:
I have a wip of this here: https://github.com/ava-labs/icm-services/pull/724

But agree that it may not be worth blocking this PR on.